packages/Collapsible/stories/examples/Showcase.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import faker from "faker";
+import StoryHeader from "storybook/components/StoryHeader";
+import { Story } from "storybook/assets/styles/common.styles";
```

Comment:
For DX it’d be nice if the story-related components could be imported from one place (e.g. `storybook` by re-exporting from an index file) so that there’s no need to remember where to import each one from

Reply:
I can fix this in a following PR tomorrow :D if possibru